servicecontrol/monitoring-instances/installation/index.md

```diff
@@ -7,33 +7,45 @@ tags:
 
 The ServiceControl installation file consists of an embedded MSI bootstrapper EXE and an embedded MSI. This installation can be executed standalone or via the [Platform Installer](/platform/installer/). The installation package include a utility to manage the installation, upgrade and remove of ServiceControl services, including Monitoring instances. This utility is launched as the final step in the installation process and is also available via the Windows Start Menu.
 
-
```

Comment:
Why was this removed? We typically want to have two spaces between sections.

Reply:
I use markdown lint and it has 1 as the default. Why would we want 2?

Reply:
Because the docs guidelines say we should have two spaces before a header: https://github.com/Particular/docs.particular.net#spaces

Reply:
Ah, didn't know that!  Still curious why we need the extra one :)

Reply:
All spacing fixed now